### ISSUE TITLE
[New rule] Avoid triple quotes when not needed

### ISSUE THREAD

**Issue by NickBalnaves** [dart-code-checker/dart-code-metrics#802]
**Please describe what the rule should do:**
Prefer single quotes instead of triple quotes if the string only spans one line.

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[x] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about (it will be better if you can provide both good and bad examples):**

Good:
```dart
Text('My text'),
```
```dart
Text(
  'My '
  'text',
),
```

Bad:
```dart
Text('''My text'''),
```
```dart
Text(
  'My '
  '''text''',
),
```

**Are you willing to submit a pull request to implement this rule?**
No
